e2e/multisig-migration.test.ts

```diff
@@ -52,6 +51,20 @@ describe("Should change LBC owner to the multisig", function () {
     ).to.not.be.reverted;
     const newLbcOwner = await lbc.owner();
     console.info("New LBC owner:", newLbcOwner);
+
+    await expect(
+      lbc.connect(impersonatedSigner).setProviderStatus(1, false)
+    ).to.be.revertedWith("LBC005");
+
+    expect(
+      multisigExecProviderStatusChangeTransaction(safeAddress, lbc)
+    ).to.eventually.be.equal(true);
+    expect(
+      multisigExecUpgradeTransaction(impersonatedSigner.address, lbc)
```

Comment:
I have a doubt about this line, if your impersonated signer is an EOA, why are you passing it as it was a multisig address? I think you should try to just upgrade like you did in line 56 with `setProviderStatus`